User can start a course

mukeshm opened the pull request:
PR for issue #8

Commit:
Initialize electron project

rsbh committed:
Moved index.html to src folder
Created style.css
Added react and its dependencies module

rsbh committed:
Setup webpack and babel for React

Commit:
Add database and models

Add nedb support
Add camo for ODM

Commit:
Add addCourse and getCourse

Add adding of course and getting of course
Add ipc listeners

rsbh committed:
Created CreateCourse and Home Component.
Changed style.css file.
Add routes to new component  in app.js

rsbh committed:
Merge branch 'development' of github.com:nilenso/dhobi-seva-electron into development

rsbh committed:
Added Validation in Create Course Form.

rsbh committed:
Added Duration Validator in CreateCourse

Commit:
Handle error callbacks

Commit:
Merge branch 'development' of github.com:nilenso/dhobi-seva-electron into development

rsbh committed:
Removed bundle.js
Added webpack script in package.json
changed button pointer

Commit:
Update README

deobald commented on src/components/CreateCourse.js:
```diff
@@ -0,0 +1,94 @@
+      if (this.state.duration === null || this.state.duration <= 0) {
+        this.setState({durationClass: 'inputText errorInput'})
+      }
+    }else {
```
The above code is pretty repetitive and a lot of nested ifs. How do you think you could make it cleaner? :)

rsbh replied:
ok, will try to make it cleaner.

rsbh replied:
@deobald All the nested ifs and repetitive code are removed.

deobald reviewed (changes requested):
Nice job! I only had one comment to make.

rsbh committed:
Created New Functions for Data Validaration
isNameValid for couse name
isDateValid for Start Date
isDuratioinValid for Course Duration

deobald reviewed (commented):
wups! I forgot to submit the review. sorry.
